lib/disc.rb

```diff
@@ -101,7 +109,8 @@ def disque=(disque)
         @disque = disque
       end
 
-      def disc(options = {})
+      def disc(queue: nil, **options)
+        @queue = queue
```

Comment:
So here's an interesting bug:

``` ruby
class Bleh
  include Disc::Worker
  disc maxlen: 1
end

p Bleh.queue #=> nil
```

This is because calling `disc` sets `@queue`, regardless of whether it's resetting it to `nil` or not. A way solution is only setting it if it's not nil. However, that's just sweeping the problem under the rug. I think this points to the API of `Disc::Job` being too smart and implicit, instead of dumb and explicit.

Not sure what's the best solution off the top of my head. @pote what do you think?